### PR TITLE
Design event form

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #60

### CODE REVIEW

File: views/event-form.hbs [hunk 1]
```diff
@@ -1,190 +1,218 @@
...
+        <td class="w-100">
+          {{#if event.ar.description}}
+            <textarea class="input-reset w-100 br2 ba b--gray" type="text" name="description_ar" value="{{event.ar.description}}"></textarea>
+          {{else}}
+            <textarea class="input-reset w-100 br2 ba b--gray dib" type="text" name="description_ar" placeholder="{{localText.arabic}}"></textarea>
+          {{/if}}
+        </td>
+    </tr>
+    <tr>
+      <td class="w-20 dark-text f4 b pa3 tr">{{localText.placeAddress}}</td>
```
Comment (m4v15) [inline views/event-form.hbs]: We don't want an address field in the event form do we? Just the place drop down list?

File: views/event-form.hbs [hunk 2]
```diff
@@ -1,190 +1,218 @@
...
 {{else}}
   <form class="" action="/{{lang}}/add-event" method="post">
-    <h1>{{localText.add}}</h1>
     <input type="hidden" name="_method" value="post">
 {{/if}}
+<table class="f6 w-100 mw8 center bb b--turquoise bw2">
+  <tbody>
+    <tr>
+      <td class="w-20 dark-text f4 b pa3 tr">*{{localText.name}}</td>
+      <td class="w-100">
```
Comment (m4v15) [inline views/event-form.hbs]: Again, change these to w-70

File: views/event-form.hbs [hunk 3]
```diff
@@ -1,190 +1,218 @@
...
+          {{#if event.en.name}}
+            <input class="input-reset w-100 br2 ba b--gray" type="text" name="name_en" value="{{event.en.name}}">
```
Comment (m4v15) [inline views/event-form.hbs]: and all of these to w-70

File: views/event-form.hbs [hunk 4]
```diff
@@ -1,190 +1,218 @@
...
+          {{else}}
+            <input class="input-reset w-100 br2 ba b--gray dib" type="text" name="name_ar" placeholder="{{localText.arabic}}">
+          {{/if}}
+        </td>
+    </tr>
+    <tr>
+      <td class="w-20 dark-text f4 b pa3 tr">{{localText.description}}</td>
+      <td class="w-100 pt3">
+          {{#if event.en.description}}
+            <textarea class="input-reset w-100 br2 ba b--gray" type="text" name="description_en" value="{{event.en.description}}"></textarea>
```
Comment (m4v15) [inline views/event-form.hbs]: same thing as in the places form - don't use value here

File: views/event-form.hbs [hunk 5]
```diff
@@ -1,190 +1,218 @@
...
+            <input class="input-reset w-100 br2 ba b--gray dib" type="text" name="address_en" placeholder="{{localText.english}}">
+          {{/if}}
+      </td>
+    </tr>
+    <tr>
+        <td class="w-20 dark-text f4 b pa3 tr">{{localText.eventsPlace}}</td>
+        <td class="w-100">
+          <select class="w-100" name="eventPlace">
+            <option value="">{{localText.selectPlace}}</option>
+            {{#each placesNameAndIds}}
```
Comment (m4v15) [inline views/event-form.hbs]: should be `placesNamesAndIds` - the drop down list doesn't work atm

File: views/event-form.hbs [hunk 6]
```diff
@@ -1,190 +1,218 @@
...
+              <option value="{{this.id}}" {{checkDropDown ../event.place._id this.id}}>{{this.name}}</option>
```
Comment (m4v15) [inline views/event-form.hbs]: and this should be `checkedDropDown`

File: views/event-form.hbs [hunk 7]
```diff
@@ -1,190 +1,218 @@
...
+    <tr>
+      <td class="w-20 dark-text f4 b pa3 tr">{{localText.accessibilityOptions}}
+      </td>
+      <td><table><tr>
+          <td class="flex items-center mb2 ">
+            <input class="mr2" id="braille" type="checkbox" name="accessibility" value="Braille" {{checkOptionsValue place.accessibilityOptions "Braille"}}>
+            <label for="braille" class="light-text">{{localText.brailleMenu}}</label>
+          </td>
+          <td class="flex items-center mb2">
+            <input class="mr2" id="audio-recordings" type="checkbox" name="accessibility" value="Audio-recordings" {{checkOptionsValue place.accessibilityOptions "Audio-recordings"}}>
```
Comment (m4v15) [inline views/event-form.hbs]: Audio-recordings again - change to `Audio recordings`

File: views/event-form.hbs [hunk 8]
```diff
@@ -1,190 +1,218 @@
...
+            <input class="mr2" id="disabled-parking" type="checkbox" name="accessibility" value="Disabled parking" {{checkOptionsValue place.accessibilityOptions "Disabled parking"}}>
+            <label for="disabled-parking" class="light-text">{{localText.disabledParking}}</label>
+          </td>
+          <td class="flex items-center mb2">
+            <input class="mr2" id="disabled-toilets" type="checkbox" name="accessibility" value="Disabled toilets" {{checkOptionsValue place.accessibilityOptions "Disabled toilets"}}>
+            <label for="disabled-toilets" class="light-text">{{localText.disabledToilet}}</label>
+          </td>
+        <td class="dark-text f4 b pa3">*{{localText.categories}}</td>
+        <td class="flex items-center mb2">
+          <input class="mr2" id="cafe" type="checkbox" name="categories" value="competition" {{checkOptionsValue event.categories "competition"}}>
```
Comment (m4v15) [inline views/event-form.hbs]: `id=cafe` should be `id=competition`

File: views/event-form.hbs [hunk 9]
```diff
@@ -1,190 +1,218 @@
...
+            <input class="input-reset w-100 br2 ba b--gray dib" type="text" name="imageUrl">
+          {{/if}}
+      </td>
+    </tr>
+    <tr>
+      <td class="w-20 dark-text f4 b pa3 tr">{{localText.accessibilityOptions}}
+      </td>
+      <td><table><tr>
+          <td class="flex items-center mb2 ">
+            <input class="mr2" id="braille" type="checkbox" name="accessibility" value="Braille" {{checkOptionsValue place.accessibilityOptions "Braille"}}>
```
Comment (m4v15) [inline views/event-form.hbs]: all of the `{{checkOptionsValue place.accessibilityOptions`

should be `{{checkOptionsValue event.accessibilityOptions`

**Review by m4v15 — CHANGES_REQUESTED** [review]
This hasn't been merged with the most up to date master yet I think

**Comment by Suuuuha** [comment]
@m4v15

**Comment by m4v15** [comment]
I was gonna say you need to change the `*this is a required field` stuff and to capitalize the categories, but I think when it's merged with master it's fine

**Review by m4v15 — APPROVED** [review]

**Pull request merged**